extensions/wikia/Recirculation/js/plista.js

```diff
@@ -0,0 +1,61 @@
+define('ext.wikia.recirculation.plista', ['jquery'], function ($) {
+	function shouldFetchPlista(items) {
+		var index = items.findIndex(function (item) {
+			return item.presented_by;
+		});
+
+		return index === -1 && ['AU', 'NZ'].indexOf(Geo.getCountryCode()) > -1;
+	}
+
+	function fetchPlista() {
+		return $.getJSON('http://farm.plista.com/recommendation/?publickey=845c651d11cf72a0f766713f&widgetname=api' +
+			'&count=1&adcount=1&image[width]=583&image[height]=328'
```

Comment:
For the other large items in footer recirc we use 320x180. Should we do the same here?